src/Support/ModularServiceProvider.php

```diff
@@ -155,41 +159,47 @@ protected function bootRoutes(): void
 			return;
 		}
 		
-		$this->autoDiscoveryHelper()
-			->routeFileFinder()
-			->each(function(SplFileInfo $file) {
-				require $file->getRealPath();
-			});
+		if (!$routes = $this->autoDiscoveryHelper()->routes()) {
+			return;
+		}
+		
+		$routes->each(function($filename) {
+			require $filename;
+		});
```

Comment:
It might be better to just pass the closure to the auto discovery helper, which could just do the check before passing it to `each()` — that would let us avoid a bunch of null checks.